Source/ACE.Server/Factories/LootHelper.cs

```diff
@@ -54,6 +54,260 @@ public static class LootHelper
                                                                 {50, "Zircon"}
                                                             };
 
+        public static int[][] ScrollSpells =
```

Comment:
In the future, maybe we should look into building these dynamically at startup from the .dat. I think that would help allow people to modify their .dat's to update the server. By doing static lists, I think that we may end up with dat mismatches if that was the case.

Also, keep in mind that there are many spells in the dat that are duplicates. They use different id's for the quest versions vs the loot generated versions.